cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,45 @@
-describe('', () => {
-  before(() => {
+/// <reference types = 'cypress' />
 
+const { ArticlePage } = require('../support/PageObjects/ArticlePage');
```

Comment:
```suggestion
const { ArticlePage } = require('../support/PageObjects/ArticlePage');
```
can't find this file in your pr, add it pls

Reply:
ArticlePage.js added with a new commit